components/FullScrenDropZone.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <v-overlay
+    v-model="UIStore.showDropZone"
+    @keyup.esc="UIStore.setShowDropZone(false)"
+    @click:outside="UIStore.setShowDropZone(false)"
```

Comment:
faut pt référencer la meme fonction, histoire de pas propager des erreurs plus tard
```suggestion
    @keyup.esc="disableDropZone"
    @click:outside="disableDropZone"
```